Makefile

```diff
@@ -9,16 +9,13 @@ SOURCEDIR     = docs/source
 BUILDDIR      = docs
 MODULEDIR     = st3
 
-# Put it first so that "make" without argument is like "make help".
-help:
-	@$(SPHINXBUILD) -M help "$(SOURCEDIR)" "$(BUILDDIR)" $(SPHINXOPTS) $(O)
-
-.PHONY: help Makefile source
+.PHONY: Makefile source clean
```

Comment:
You can remove `Makefile` as it isn't used as a dependency for a target anymore.